### PR TITLE
feat: support for humann3 utilities

### PR DESCRIPTION
**Pull request by bcai2**
Adds support for the following utilities:
- humann_barplot
- humann_gene_families_genus_level
- humann_join_tables
- humann_reduce_table
- humann_regroup_table
- humann_renorm_table
- humann_split_stratified_table
- humann_unpack_pathways

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
             cd ~/repo
             export DEPLOY_ENVIRONMENT=$([[ $CIRCLE_BRANCH = main ]] && echo production || echo staging)
             shopt -s globstar
-            TESTFILES=$(circleci tests glob tests/**/test*.py | circleci tests split --split-by=filesize)
+            TESTFILES=$(circleci tests glob tests/**/test*.py | circleci tests split --split-by=timings)
```
Comment (bcai2) [inline .circleci/config.yml:61]: 👍

File: tests/test_humann3.py [hunk 2]
```diff
@@ -33,11 +33,12 @@ def test_humann3_m8():
...
 
 
-@pytest.mark.integration_full
+@pytest.mark.integration
 def test_humann3_fastq():
     """
     Tests humann3 with a fastq.gz file
-    Note: This test takes about 1 hour and should be run on pre-deploy tests only.
+    Note: This test uses taxonomic profile (from running the same input without one) to speed up execution. This skips
+    some steps that would otherwise happen but cause the test to take around an hour.
```
Comment (bcai2) [inline tests/test_humann3.py:41]: How long does the new test take?

**Review by bcai2 — COMMENTED** [review]
LGTM, just a quick question

File: tests/test_humann3.py (hunk 2 above)
Comment (jherr-dev) [reply tests/test_humann3.py:41]: Took < 5 minutes if I remember correctly from last week.

**Review by lebovic — APPROVED** [review]

**Pull request merged**